enterprise/cypher/cypher/src/test/scala/org/neo4j/cypher/ExecutionEngineTest.scala

```diff
@@ -40,7 +40,7 @@ import org.neo4j.kernel.impl.coreapi.TopLevelTransaction
 import org.neo4j.test.{TestEnterpriseGraphDatabaseFactory, TestGraphDatabaseFactory}
 
 import scala.collection.JavaConverters._
-import scala.collection.mutable
+import scala.collection.{Map, mutable}
```

Comment:
This appears to change the type used for the overriding `databaseConfig()` method, such that this class no longer compiles.